CMakeFilters.cmake

```diff
@@ -9,12 +9,12 @@
 # If you do not have access to either file, you may request a copy from
 # help@hdfgroup.org.
 #
-cmake_dependent_option (HDF5_USE_ZLIB_NG "Use zlib-ng library as zlib library" OFF HDF5_ENABLE_ZLIB_SUPPORT OFF)
+option (HDF5_USE_ZLIB_NG "Use zlib-ng library as zlib library" OFF)
 cmake_dependent_option (HDF5_USE_ZLIB_STATIC "Find static zlib library" OFF HDF5_ENABLE_ZLIB_SUPPORT OFF)
 cmake_dependent_option (HDF5_USE_LIBAEC_STATIC "Find static AEC library" OFF HDF5_ENABLE_SZIP_SUPPORT OFF)
-cmake_dependent_option (ZLIB_USE_EXTERNAL "Use External Library Building for ZLIB" OFF HDF5_ENABLE_ZLIB_SUPPORT OFF)
+option (ZLIB_USE_EXTERNAL "Use External Library Building for ZLIB" OFF)
```

Comment:
I see that HDF5_ was added to some for consistency but there are still some without it, such as this one, and there are some still have H5_, like H5_ZLIB_FOUND.  Is there a pattern/logic?